src/app/features/vendor/vendor-listing/vendor-listing.component.ts

```diff
@@ -32,7 +33,7 @@ import { IVendor } from '../models/vendor.model';
 })
 export class VendorListingComponent implements OnInit {
   vendors!: IVendor[];
-
+ constructor(private router : Router){}
```

Comment:
add empty line before and after constructor

Reply:
done